test/evergreen.yml

```diff
@@ -134,6 +149,16 @@ functions:
         for i in $(seq ${times|1}); do
           ./t -1 -c ${config|../../../test/format/CONFIG.stress} ${extra_args|}
         done
+  "format test script":
+    command: shell.exec
+    params:
+      working_dir: "wiredtiger/test/format"
+      script: |
+        set -o errexit
+        set -o verbose
+        for i in $(seq ${times|1}); do
+          ${test_env_vars|} ./format.sh ${smp_command|} ${format_test_script_args|} 2>&1
+        done
```

Comment:
This new function seems to be introduced in the PR of WT-5000 as well. Depending on which ticket is merged first, the other ticket may want to drop this new function definition. @raviprakashgiri29 @hbokhari can I leave it to you to coordinate the 2 PR merges.

Reply:
Yes, I used this function just for my testing to progress. Merged the latest changes by @hbokhari.